airbyte-cdk/python/airbyte_cdk/sources/declarative/async_job/job.py

```diff
@@ -4,8 +4,8 @@
 from datetime import timedelta
 from typing import Optional
 
-from airbyte_cdk import StreamSlice
 from airbyte_cdk.sources.declarative.async_job.timer import Timer
+from airbyte_cdk.sources.types import StreamSlice
```

Comment:
For my personal knowledge: Is there a reason you changed that? I mean, I prefer it as I've had issues in the past with partially initialized modules but I didn't have that when we introduced the async components

Reply:
I encountered a circular import error during testing.